content/en/docs/concepts/cluster-administration/system-logs.md

```diff
@@ -231,6 +231,53 @@ Similar to the container logs, you should rotate system component logs in the `/
 In Kubernetes clusters created by the `kube-up.sh` script, log rotation is configured by the `logrotate` tool.
 The `logrotate` tool rotates logs daily, or once the log size is greater than 100MB.
 
+## Log query
+
+{{< feature-state for_k8s_version="v1.27" state="alpha" >}}
+
+To help with debugging issues on nodes, Kubernetes v1.27 introduced a feature that allows viewing logs of services
+running on the node. To use the feature, ensure that the `NodeLogQuery`
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/) is enabled for that node, and that the
+kubelet configuration options `enableSystemLogHandler` and `enableSystemLogQuery` are both set to true. On Linux
+we assume that service logs are available either via journald. On Windows we assume that service logs are available
+in the application log provider. On both operating systems, logs are also available by reading files within
+`/var/log/`.
+
+A node level administrator can try out this alpha feature on all their nodes, or on just a subset. Provided you're
+authorized to do so, you can now query logs for system services on that node:
```

Comment:
This message is unclear to me.
"node level administrator" means nothing in the context of Kubernetes.
The authorization we can enforce is about the access to the Kubernetes API.
It sounds like we are having a feature enabled by one group of people (node-level admins),
but used by another group of people (authorized using RBAC, with no dedicated built in roles).
Am I understanding this correctly?

Reply:
I was just going by [this](https://github.com/kubernetes/website/pull/40003#discussion_r1137249798) suggestion. I have however updated it to clarify that we only enforce access to node objects.